docs/asciidoc/static/configuration.asciidoc

```diff
@@ -258,14 +271,14 @@ output {
 [float]
 === Conditionals
 
-Sometimes you only want a filter or output to process an event under
-certain conditions. For that, you'll want to use a conditional!
+Sometimes you only want a plugin to process an event under
```

Comment:
Maybe "When filtering or outputting events, you may want to make decisions about how to act on certain events. For that, you can use a conditional" ?